tests/test_interface.py

```diff
@@ -195,3 +195,21 @@ def test_power_down_can_be_switched(current_value, expected_value):
     interface_state.power_down = current_value
     interface_state.switch_power_down()
     assert interface_state.power_down is expected_value
+
+
+def test_change_callback(capsys):
+    """
+    Test if change_callback function works and assert correct result.
+    Test function "test_print" is used.
+    """
+    interface_state = start_interface_state(test_print,program=True)
```

Comment:
funkce `start_interface_state` bere dva pojmenovane argumenty: `program` a `confirmed`. Jakou roli zde plni `test_print`?

Reply:
žádnou, to jsem tam dala omylem  :-)